release_notes_generator/utils/decorators.py

```diff
@@ -41,9 +48,12 @@ def wrapped(*args, **kwargs) -> Optional[Any]:
 
 
 # pylint: disable=broad-except
-def safe_call_decorator(rate_limiter: GithubRateLimiter):
+def safe_call_decorator(rate_limiter: GithubRateLimiter) -> Callable:
     """
     Decorator factory to create a rate-limited safe call function.
+
+    @param rate_limiter: The rate limiter to use.
+    @return: The decorator.
     """
 
     def decorator(method: Callable) -> Callable:
```

Comment:
I am not sure if we also want to have a doc for these decorator methods. It is up to debate. Same in GithubRateLimiter and Factory.

Reply:
Why not. It simply says, what is that ugly function? In the future, that can be expanded as needed.